tests/terraform-tests.failures.txt

```diff
@@ -4,4 +4,5 @@ TestAccAWSEc2TransitGatewayRouteTableAssociation
 TestAccAWSEc2TransitGatewayVpcAttachment
 TestAccAWSFms
 TestAccAWSIAMRolePolicy
-TestAccAWSSecurityGroup_forceRevokeRules_
\ No newline at end of file
+TestAccAWSSecurityGroup_forceRevokeRules_
+TestAccAWSSSMDocument_package
```

Comment:
Sadly this is the only test in the `TestAccAWSSSMDocument` that is not passing. I'll work on a further PR for this test so the list of success can be just replaced with the complete name.